### PR TITLE
Add Banner message to page header, and #419 text changes

### LINKED ISSUE #393: UI: Display Banner Message
- UI update to display BannerMessage objects. 
- Add banner fixture with instructions on page including warning not to upload private data
   - similar to note on: https://demo.dataverse.org/

### LINKED ISSUE #419: (text changes) DP Creator change list
### 1. Informational/Warning Banner in Header
- [x] #420

### 2. Application-wide text changes
- [x] Restore text CSV #426
- [x] Change “dataset” to “data file” throughout the application

### 3. Page: “Home” (before login)
- #453

### 4. Page: Validate Dataset
- https://github.com/opendp/dpcreator/issues/455

### 5. Page: Confirm Variables
- [x] https://github.com/opendp/dpcreator/issues/424
- [ ] Update text instructions at the top of page

### 6. Page: Set Epsilon Value
- #456

### 7. Create the Statistics Page
- #460

### 8. Pop-up: Create Your Statistic
- [x] #437 

### 9. Pop-up: Please confirm the epsilon and delta values
- Miro reference: https://miro.com/app/board/o9J_kj6tycQ=/?moveToWidget=3074457366776798016&cot=14
- [x] Remove link: “Modify this selection”
- [x] Fix text under the Epsilon and Delta input boxes
- Information box that reads “The level of privacy”
	- [x] Remove it

### 10. Page: Generate DP Release
- Miro reference: https://miro.com/app/board/o9J_kj6tycQ=/?moveToWidget=3074457366776797824&cot=14
- [x] Bottom of page: next to the “Submit Statistics” add a “Return to Create Statistics Page” button

### 11. Page: Release Detail
#459

### LINKED ISSUE #455: Validate Dataset Text Changes
- Miro reference: https://miro.com/app/board/o9J_kj6tycQ=/?moveToWidget=3074457366777335859&cot=14
- [x] Change text  “Used **dataset**: Replication …..” to “Used **data file**”. [boldface only added to emphasize change, e.g. don’t use boldface]
- [x] Initial sentence, remove “Firstly, we need to”, start with “Confirm…”
- [x] Remove warning box that reads “You must complete this questionnaire…”
- Question: Which of the following best describes your dataset
	- 1st Choice: “Public Information”
		- [x] Add additional sentence about how DP isn’t needed
	- Last Choice: “Information that would cause sever harm…”
		- [x] Add additional sentence about how application not secure enough
	- [x] Link “data classification examples” to:
		*  https://policy.security.harvard.edu/view-data-security-level
- Pop-up: The dataset [data file] should not be used in DP Creator
	- [x] Change “DPcreator” to “DP Creator”

### LINKED ISSUE #456: Epsilon Page Text Changes
- Miro reference: https://miro.com/app/board/o9J_kj6tycQ=/?moveToWidget=3074457366776975305&cot=14
- Change page title. 
	- For future: move these questions to the “Validate Dataset” page
- Question “Is your data a secret and simple random sample from a larger population?”
	- [x] Typo in the information box, `instituions` should be `institutions`
	- [x] Add context -> if the data is a simple random sample, we can use methods (amplification) to increase the accuracy and utility of the statistics you create.
- Question 2: about dataset size being made public
     - [x] Add context: If the dataset size can be made public, less of the privacy budget is used when creating statistics

### LINKED ISSUE #458: Error messages on the Create Statistics pop up
image
image

### LINKED ISSUE #459: Release Page Text Changes
- Miro reference: https://miro.com/app/board/o9J_kj6tycQ=/?moveToWidget=3074457366776693470&cot=14
- [x] Remove HTML download button
- [x] Remove XML download button
- Link “Check DP release in Dataverse”
	- [x] Link appropriately (back to the Dataset page or DV download file)
- Table with “DV Installation”, “Remaining time to complete release”, etc.
	- [x] Move this table after "DP Release Information" and before the accordion panels
	- [x] If the process is complete, remove the row "Remaining time to complete release"
	- ref: https://miro.com/app/board/o9J_kj6tycQ=/?moveToWidget=3074457368233364946&cot=14
- [x] for support email at the bottom, use `info@opendp.org`

### PR DESCRIPTION
**Pull request by ekraffmiller**

### CODE REVIEW

File: client/src/components/Structure/Header.vue [hunk 1]
```diff
@@ -128,11 +144,24 @@
...
+  components: {ColoredBorderAlert},
+  created() {
+    this.$store.dispatch('auth/fetchBannerMessages')
+  },
   computed: {
     ...mapGetters('auth', ['isAuthenticated', 'isTermsAccepted']),
-    ...mapState('auth', ['user']),
+    ...mapState('auth', ['user', 'bannerMessages']),
+    bannerMessage() {
+      if (this.bannerMessages) {
```
Comment (raprasad) [inline client/src/components/Structure/Header.vue:163]: The UI should be able to display multiple banner messages--the API will only return active banner messages

**Review by raprasad — CHANGES_REQUESTED** [review]
please see review

File: client/src/components/Structure/Header.vue (hunk 1 above)
Comment (ekraffmiller) [reply client/src/components/Structure/Header.vue:163]: I just pushed a change to display a list of messages

Comment (ekraffmiller) [reply client/src/components/Structure/Header.vue:163]: I just realized I removed the condition for displaying the box - need to do one more change

File: client/src/components/Structure/Header.vue [hunk 2]
```diff
@@ -59,6 +59,26 @@
...
+        <v-spacer></v-spacer>
+        <v-container>
+          <v-row>
+            <v-col cols="12">
+
+              <ColoredBorderAlert type="warning">
+                <template v-slot:content>
+                  <ul style="list-style-type:none;">
+                    <li v-for="message in bannerMessages">
+                      <div v-html="message.content"></div>
```
Comment (raprasad) [inline client/src/components/Structure/Header.vue:72]: Add some kind of "spacer"/mark between messages. e.g. something like:

```html
                   <li v-for="(message, index) in bannerMessages">
                      <div v-html="message.content"></div>
                        <hr v-if="index != Object.keys(bannerMessages).length - 1" style="margin:10px 0; color:#cccccc;" />
                    </li>
```

**Review by raprasad — DISMISSED** [review]

**Review by raprasad — APPROVED** [review]
approved

**Pull request merged**